src/controller/expense.controller.js

```diff
@@ -0,0 +1,73 @@
+const expenseService = require('../service/expenses.service');
+
+const getExpenses = (req, res) => {
+  const expense = expenseService.getExpenses(req.query);
+
+  if (!expense) {
+    return res.status(404).json({ message: 'NOT FOUND' });
+  }
+
+  return res.status(200).json(expense);
+};
+
+const createExpense = (req, res) => {
+  const newExpense = expenseService.createExpense(req.body);
+
+  if (newExpense === null) {
+    return res.status(400).json({
+      message:
+        "'userId','title','amount' and 'category' content MUST BE filled",
+    });
+  }
+
+  if (newExpense === undefined) {
```

Comment:
For practical reasons, is this case different than the first one? (newExpense === null)